site/content/en/docs/Guides/_index.md

```diff
@@ -5,5 +5,3 @@ weight: 4
 description: >
   Guides on using Open Match
 ---
-
-{{< pagelist >}}
```

Comment:
Whats the motivation to cleanup pagelist? Wont that leave us with completely blank intro pages? Might be more appropriate to add better descriptions to the pages so pagelist view becomes more meaningful?